swift/downgrades/44e36e15e90bc1535964d9b86b3cd06a8b0d26e3/upgrade.properties

```diff
@@ -0,0 +1,2 @@
+description: Revert removing `getInterpolationCountExpr` and `getLiteralCapacityExpr` from `InterpolatedStringLiteralExpr`
+compatibility: full
```

Comment:
So the downgrade is to do nothing, giving us an empty `interpolated_string_literal_expr_interpolation_count_exprs` and `interpolated_string_literal_expr_literal_capacity_exprs` (and no associated integer literal expressions).  I see that we don't actually use these in queries, but they were exposed (e.g. by `getInterpolationCountExpr()`), so I think perhaps `backwards` compatibility might be more accurate for the downgrade script???

Reply:
100% agree this is not super important, and shouldn't hold up merging the PR.

Reply:
changed to backwards :+1: